File: vdp-php/tests/vctc/ProgramAdministrationTest.php
<?php

namespace Vdp;

class ProgramAdministrationTest extends \PHPUnit_Framework_TestCase {
	
	public function setUp() {
		$this->abstractVisaAPIClient = new AbstractVisaAPIClient;
	}
	
	public function testRetreiveTransactionTypeControls() {
		$baseUrl = "vctc/";
		$resourcePath = "programadmin/v1/configuration/transactiontypecontrols";
		$statusCode = $this->abstractVisaAPIClient->doMutualAuthCall ( 'get', $baseUrl.$resourcePath, 'Retrieve Transaction Type Control call', '' );
		$this->assertEquals($statusCode, "200");
	}
}

File: vdp-c-sharp/vdp-c-sharp/VDPApiTest/vctc/ProgramAdministrationTest.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace Vdp
{
    [TestClass]
    public class ProgramAdministrationTest
    {
        private AbstractVisaAPIClient abstractVisaAPIClient;

        public ProgramAdministrationTest()
        {
            abstractVisaAPIClient = new AbstractVisaAPIClient();
        }

        [TestMethod]
        public void TestRetreiveTransactionTypeControls()
        {
            string baseUri = "vctc/";
            string resourcePath = "programadmin/v1/configuration/transactiontypecontrols";
            string status = abstractVisaAPIClient.DoMutualAuthCall(baseUri + resourcePath, "GET", "Retreive Transaction Type Controls Test", "");
            Assert.AreEqual(status, "OK");
        }
    }
}
